### PR TITLE
2024-05-23 minutes 

### PR DESCRIPTION
**Pull request by planetf1**
Adds TSC minutes from 2024-05-23

TSC members - please review & raise any comments/approve -- thanks

### CODE REVIEW

**Comment by planetf1** [comment]
@ryjones does our zoom account record an attendee list. I wasn't able to see alongside the recording. I can see everyone who spoke, but not any other attendees.

**Review by dstebila — APPROVED** [review]

**Review by mkannwischer — APPROVED** [review]

**Review by mbbarbosa — APPROVED** [review]

**Comment by planetf1** [comment]
Thanks @ryjones - updated attendees from zoom record.

File: meetings/2024-05-23/minutes.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+A security issue was also reported which now has a patch.
+
+Ry pointed out that OQS has a [dashboard](https://openquantumsafe.org/dashboard) which can include links to OpenSSF reports. Nigel mentioned if we benchmark this could be useful info to add. Max pointed out we need to ensure the content is actionable, ie we use it.
+
+### What does assured mean
+
+We reviewed the discussion so far in [mlkem-c-aarch64#37](https://github.com/pq-code-package/mlkem-c-aarch64/issues/37)
+
+Minimum could be to run tests properly, use test vectors, run static analysis. If going for higher assurance we should do more - ie formal methods to prove no undefined behaviour, constant-time verification. Later extending to assembly correctness. Maybe we also look into additional tooling like CBMC especially when working on highly micro-optimized assembly.
```
Comment (hanno-becker) [inline meetings/2024-05-23/minutes.md:101]: CBMC would be useful for absence of undefined behavior in C, and perhaps some mild functional assurances. 
```suggestion
Minimum could be to run tests properly, use test vectors, run static analysis. If going for higher assurance we should do more - ie formal methods to prove no undefined behaviour (e.g. using CBMC), constant-time verification. Later extending to assembly correctness. 
```

File: meetings/2024-05-23/minutes.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+'production-ready'- very dependent on user's requirements ie embedded system vs cloud-scale service. Other aspects - like supply-chain security, performance, portability, BOMs are part of this too. Different implementations will vary (ie libjade etc is higher). Maybe alpha/pre-releases have lower guarantees. What's needed is to explain and document. can raise the bar over time.
+
+Manuel clarified a thought early on was to establish a taxonomy - or at least a set of terms that describe the levels of assurance.
+
+Rod pointed out that other industry assurance cases (safety critical etc) go with 'principles based assurance', or 'evidence based assurance' whereby you make a claim, and provide evidence to support - that may include tool used. Also clearly state assumptions. We should aim-high as seen with libjade, or (proprietary) AWS.
```
Comment (hanno-becker) [inline meetings/2024-05-23/minutes.md:107]: The AWS work Rod was referring to is https://github.com/awslabs/s2n-bignum which is an open source (+ open proof) library of formally verified cryptographic primitives. 
```suggestion
Rod pointed out that other industry assurance cases (safety critical etc) go with 'principles based assurance', or 'evidence based assurance' whereby you make a claim, and provide evidence to support - that may include tool used. Also clearly state assumptions. We should aim-high as seen with libjade, or AWS' s2n-bignum.
```

**Review by hanno-becker — COMMENTED** [review]
Thanks @planetf1! Suggested a few minor clarifications.

File: meetings/2024-05-23/minutes.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+* Review  [open TSC issues](https://github.com/pq-code-package/tsc/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc):
+
+  * any notable progress
+  * topics for next meeting
+* Any other business
+* Next meeting / ongoing schedule
+
+## Introductions
+
+Rod Chapman introduced himself as a colleague of Hanno's at AWS Cryptography Group with a particular interest in formal verification of PQC. He has previously build a formally verified implementation of Kyber.
```
[the remark this answers is not in the document]
Comment (planetf1) [reply meetings/2024-05-23/minutes.md:39]: thanks for the link. added

**Review by hanno-becker — APPROVED** [review]

**Comment by planetf1** [comment]
I will now merge these as we have sufficient approvals.
Any further corrections, please raise a new issue.

**Pull request merged**